### ISSUE TITLE
INAV 2.2.0  RANGEFINDER=UNAVAILABLE

### ISSUE THREAD

**Issue by lugica2001** [iNavFlight/inav-rangefinder#8]
INAV_I2C does not work on MATEKF722. 

I followed the suggestion in 
https://github.com/iNavFlight/inav-rangefinder/issues/4
bust RANGEFINDER is UNAVAILABLE

I checked with the oscilloscope and the FC does not send anything at the address 0x14.

**Comment by richard0hidvegi** [comment]
> 
> 
> INAV_I2C does not work on MATEKF722.
> 
> I followed the suggestion in
> #4
> bust RANGEFINDER is UNAVAILABLE
> 
> I checked with the oscilloscope and the FC does not send anything at the address 0x14.

Hello,

I am working with a similar setup. I am using a 5V nano with level converters. I noticed a strange behavior on my system. If I power up the arduino first and after starting a main board the FC starting the communication. It might be just an isolated incident but maybe it can help. I checked the arduino part and it seems to be the problematic part for me.
